### PR TITLE
Update version_5_upgrade.html.markdown

### PR DESCRIPTION
**Pull request by hao-nan-li**
https://github.com/GoogleCloudPlatform/magic-modules/pull/8896

**Release Note Template for Downstream PRs (will be copied)**

```release-note:none
```

### CODE REVIEW

File: mmv1/third_party/terraform/website/docs/guides/version_5_upgrade.html.markdown [hunk 1]
```diff
@@ -437,4 +437,10 @@ resource "google_project_iam_binding" "gcs-bucket-writer" {
...
 
-`google_service_networking_connection` now uses the Create endpoint instead of the Patch endpoint during the creation step. Previously, Patch was used as a workaround for an issue that has since been resolved.
\ No newline at end of file
+`google_service_networking_connection` now uses the Create endpoint instead of the Patch endpoint during the creation step. Previously, Patch was used as a workaround for an issue that has since been resolved.
+
+## Resource: `google_container_cluster`
+
+### `gce_persistent_disk_csi_driver_config` now is enabled by default on newly created clusters
+
+Previously the doc stated that `gce_persistent_disk_csi_driver` defaults to `disable`, now it defaults to `enable`.
```
Comment (c2thorn) [inline mmv1/third_party/terraform/website/docs/guides/version_5_upgrade.html.markdown:446]: are you referring to API documentation or our documentation? What about just "Previously  `gce_persistent_disk_csi_driver` defaulted to `disable` ..."

Comment (hao-nan-li) [reply mmv1/third_party/terraform/website/docs/guides/version_5_upgrade.html.markdown:446]: Updated our documentation with the API documentation. Yeah nice catch on the past tense.

Comment (c2thorn) [reply mmv1/third_party/terraform/website/docs/guides/version_5_upgrade.html.markdown]: I misunderstood, so this is a doc-only change? A doc-only change shouldn't need an upgrade guide or really even be in 5.0.0 at all. There would be nothing breaking

Comment (c2thorn) [reply mmv1/third_party/terraform/website/docs/guides/version_5_upgrade.html.markdown]: Reading from https://github.com/hashicorp/terraform-provider-google/issues/10035 I see the context. Are we aligned with the API behavior? If not, then we should change the default and that would be breaking.

If we are already aligned, then the documentation can just change in `main`. don't even need a changelog.

Comment (hao-nan-li) [reply mmv1/third_party/terraform/website/docs/guides/version_5_upgrade.html.markdown]: In the ticket, it said our doc says `gce_persistent_disk_csi_driver_config` defaults to `disabled`. Here I don't see a default value https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/third_party/terraform/services/container/resource_container_cluster.go.erb#L377.

Also we currently require the enable field. If the API is set to `enable` by default, we don't necessarily need to mark this field as `required`.

Comment (c2thorn) [reply mmv1/third_party/terraform/website/docs/guides/version_5_upgrade.html.markdown]: Let's go with option 2. That means we can close this PR and just have https://github.com/GoogleCloudPlatform/magic-modules/pull/8896 be in `main`.

**Pull request closed without merging**